### PR TITLE
emacsPackages.idris2-mode: updateScript

### PR DESCRIPTION
**Pull request by AndersonTorres**
## Description of changes

https://github.com/NixOS/nixpkgs/issues/278925
https://github.com/NixOS/nixpkgs/pull/329005

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by AndersonTorres** [comment]
@wuyoli

**Comment by AndersonTorres** [comment]
I suspect wuyoli is inactive...

**Comment by AndersonTorres** [comment]
@jian-lin I added another commit transferring the maintenance of this package to me, since @wuyoli does not seem active.

File: pkgs/applications/editors/emacs/elisp-packages/manual-packages/idris2-mode/default.nix [hunk 1]
```diff
@@ -20,10 +21,12 @@ melpaBuild rec {
...
   ];
 
+  passthru.updateScript = gitUpdater { };
+
   meta = {
     homepage = "https://github.com/idris-community/idris2-mode";
     description = "Emacs mode for editing Idris 2 code";
     license = lib.licenses.gpl3Only;
-    maintainers = with lib.maintainers; [ wuyoli ];
+    maintainers = with lib.maintainers; [ AndersonTorres ];
```
Comment (jian-lin) [inline pkgs/applications/editors/emacs/elisp-packages/manual-packages/idris2-mode/default.nix:30]: What about co-maintaining?

From your commit message:
> wuyoli is inactive from at least one year

They created a [PR] last month.  I am a bit confused about "at least one year".



[PR]: https://github.com/NixOS/nixpkgs/pull/323443

Comment (AndersonTorres) [reply pkgs/applications/editors/emacs/elisp-packages/manual-packages/idris2-mode/default.nix:30]: I was looking at the Git commit logs. Theey has only four commits: 

- one adding theemself as maintainer
- this package
- a fixup on river
- a fixup on avizo: 
```
  cce4df2373afdff8f9c53bd78841bc01cea474c3
  CommitDate: Sat Mar 18 00:56:49 2023 +0100
```

Comment (AndersonTorres) [reply pkgs/applications/editors/emacs/elisp-packages/manual-packages/idris2-mode/default.nix]: Co-maintainership is OK, tho.

**Review by jian-lin — APPROVED** [review]

**Pull request merged**